### ISSUE TITLE
The name `main_color' does not exist in the context of `LiveChart.Drawable'

### ISSUE THREAD

**Issue by stsdc** [lcallarec/live-chart#25]
So I tried to compile Monitor with new 1.9.0 and am having this error. I know it's deprecated, but it still should compile right?
If I comment setting background colour everything compiles.

```
../src/Widgets/Chart/Chart.vala:30.9-30.40: error: The name `main_color' does not exist in the context of `LiveChart.Drawable'
        live_chart.background.main_color = Gdk.RGBA () {
```
```vala
        live_chart = new LiveChart.Chart (config);
        live_chart.expand = true;
        live_chart.legend.visible = false;
        live_chart.grid.visible = true;
        live_chart.background.main_color = Gdk.RGBA () {
            red = 1, green = 1, blue = 1, alpha = 1
        }; // White background
```

**Comment by lcallarec** [comment]
Damn !

My bad, it's fixed in [1.9.1](https://github.com/lcallarec/live-chart/releases/tag/1.9.1).

(However, I suggest that you should take care to remove deprecated method invocation, Livechart 2.0 will be released in a few months and every methods marked as deprecated will be removed. The good news is that v2 won't break anything but deprecated)

**Comment by stsdc** [comment]
Already did! It was just initial testing 😊️
